sat/cli/status/main.py

```diff
@@ -25,6 +25,7 @@
 Entry point for the status subcommand.
 """
 import logging
+from oauthlib.oauth2 import TokenExpiredError, InvalidGrantError
```

Comment:
I think you only need to import `InvalidGrantError`

```suggestion
from oauthlib.oauth2 import InvalidGrantError
```